runtime/examples/extension_with_esm/main.rs

```diff
@@ -30,6 +30,5 @@ async fn main() -> Result<(), AnyError> {
     },
   );
   worker.execute_main_module(&main_module).await?;
-  worker.run_event_loop(false).await?;
```

Comment:
Removed this because it `execute_main_module`  seems to already call `run_event_loop` and the second one is not needed. Please tell be if I am missing something